unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx

```diff
@@ -1,88 +0,0 @@
-import * as React from 'react';
```

Comment:
👋  Bye-bye - Going to miss you, but not going to miss all the confused end-users!